devutils/update_submodule.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+UNGOOGLED_REPO=$(dirname $(dirname $(readlink -f ${BASH_SOURCE[0]})))/ungoogled-chromium
+
+CHROMIUM_VERSION=$(curl -sL https://raw.githubusercontent.com/Eloston/ungoogled-chromium/master/chromium_version.txt)
```

Comment:
Could we just `git show --git-dir=$UNGOOGLED_REPO/.git master:chromium_version.txt` after we initialize and update the submodule?

Reply:
I actually ran a command very similar to that in Git Bash just now:

`git show --git-dir=./ungoogled-chromium/.git master:chromium-version.txt`

This is what I got for a response:

`fatal: Path 'chromium-version.txt' does not exist in 'master'`

I'm not sure where you're going with this. Are we trying to find a better way to pull the Chromium version from `ungoogled-chromium`?

Reply:
Yeah, I'm trying not to hardcode GitHub URLs, which will make things easier for mirrors and such.

I accidentally put the arguments in the wrong order. This one works: `git --git-dir=ungoogled-chromium/.git show origin/master:chromium_version.txt`

(It also uses `origin/master` instead of `master`, in case a local branch isn't created on submodule init or update.)